### DISCUSSION TITLE
Barracuda runs LSTM model and returns different values

### DISCUSSION THREAD

**Discussion by steven90311** [community/community#63638] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I am working with ai model(onnx) on Unity.
I have installed barracuda 3.0.0 package in the project.
But the output of the model(onnx) is diffent with the output of the model on Python.

**Comment by MayaAtwan** [comment]
I found this,
https://stackoverflow.com/questions/62698346/unexpected-model-output-running-onnx-model-in-unity-using-barracuda?answertab=trending#tab-top

I hope it helps.

**Reply by steven90311** [reply]
thanks for your answer.
but I have tried this solution. it's not working with that.

**Comment by eddytalvala** [comment, marked as answer]
Barracuda 3.0 doesn't support all type of LSTM.
Please check this link.
https://docs.unity.cn/Packages/com.unity.barracuda@3.0/manual/SupportedOperators.html#LSTM:~:text=Barracuda%20op%3A%20LRN-,LSTM,-ONNX%20specification

As you can see Barracuda allows only ML-Agents models.
